site/en/docs/devtools/css/reference/index.md

```diff
@@ -343,6 +343,20 @@ To add a class to an element:
 
 **Figure 14**. The **Element Classes** pane
 
+### Emulate light and dark theme preferences and enable automatic dark mode {: #emulate-light-dark-themes }
+
+To toggle automatic dark mode or emulate the user's preference of light or dark themes:
+
+1. On the **Elements** > **Styles** pane, click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/CNwUGOgogDCqUaQGt9ZS.svg", alt="Toggle common rendering emulations", width="20", height="20" %}**Toggle common rendering emulations**.
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/2G2TqgXtPcUnSgRN4saG.png", alt="Toggle common rendering emulations", width="800", height="376" %}
+1. Select one of the following from the drop-down list:
```

Comment:
```suggestion
1. On the **Elements** > **Styles** pane, click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/CNwUGOgogDCqUaQGt9ZS.svg", alt="Toggle common rendering emulations.", width="20", height="20" %}**Toggle common rendering emulations**.
   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/2G2TqgXtPcUnSgRN4saG.png", alt="Toggle common rendering emulations.", width="800", height="376" %}
1. Select one of the following from the drop-down list:
```

Reply:
Tiny nit but the style guide suggests to use punctuation in alt text https://developers.google.com/style/images?hl=en#alt-text